README.md

```diff
@@ -185,8 +185,16 @@ export class AppComponent implements AfterViewInit {
 
 ## Contribute
 Firstly fork this repo, then clone it and go inside the root of the freshly forked project.
+
+### Install
+`yarn`
+
+then
+
+`yarn add --dev node-sass`
```

Comment:
I don't understand why this line is in the readme. If `node-sass` is needed why don't add it to dependencies?

Reply:
True. @maximelafarie can you confirm we can add node-sass package in dev dependencies without any problem ?

Reply:
Removed until response, this is not related to the current PR

Reply:
What was this line? Was it added in the @melicerte's commit? Normally, nothing special has to be added in the `package.json` of the library. 🤔

Reply:
I removed it from commit.

But node-sass must be installed indeed to make demo work. To address in another issue I guess.